infrastructure/aws/mkscratch.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+set -x # Show commands
+set -eu # Errors/undefined vars are fatal
+set -o pipefail # Check all commands in a pipeline
+
+# Create the "index-scratch" directory where each specific tree's indexing
```

Comment:
This is a straight move from the other file, plus header decoration stuff (hashbang and `set` commands)